product-features/self-service-listing.md

```diff
@@ -132,15 +132,29 @@ Recommended to track transaction status:
 
 </details>
 
-### Step 2: Contract creation
+### Step 2: LP Lock & Burn Settings
+
+After submitting the Self-Service Listing Pool, a pop-up will appear, allowing the creator to lock or burn the LP tokens, or to leave the liquidity pool unlocked. By default, the Self-Service Listing Pool is set to be locked for 6 months, as it is the recommended option. Users are prompted to select one of three settings:
+
+- **Do not lock LP 🔓**: There will be no lock-up period and the initial liquidity provider will receive the corresponding LP tokens once the pool is live and operational. Since the pool is unlocked, the owner will be able to withdraw liquidity at any time. 
+
+- **LP is locked for 6 months 🔒** : This is the default option. It locks liquidity within decentralized smart contracts for a 6 month period, requiring a manual LP claim after maturity. This prevents unauthorized withdrawals and protects liquidity providers from rug pulls.
```

Comment:
1. This is the _recommended_option?
2. It locks liquidity within _ALEX_ smart contracts for a 6 month period. _When the period concludes, the pool owner can claim LP tokens (withdraw liquidity) as any other provider._ (O algo así +/- GPT)
3. Algo para reemplazar "unauthorized"?

Reply:
1. Si, esa opcion tiene una etiqueta que dice "recommended."
3. Te parece unexpected withdrawals? Sino dejo la parte de rug pull nada mas